chart/sonarqube/Chart.yaml

```diff
@@ -22,7 +22,7 @@ appVersion: 1.16.0
 
 dependencies:
   - name: sonarqube
-    version: 9.6.3
+    version: 9.10.3
     repository: https://oteemo.github.io/charts
```

Comment:
This helm registry has been deprecated. We need to move to the SonarSource version - https://github.com/SonarSource/helm-chart-sonarqube/tree/master/charts/sonarqube . Their latest helm chart is v4.0.3 . The chart is based on the Oteemo chart we have been using so hopefully the migration is straight-forward